File: S_20190727.cs
/* 
*  DB Script Tool
*  CSharp (C#) - 2019-07-27 11:03:56
*  
*  MODEL CLASSES FOR S DATABASE
*/



/* S.cs -------------------------------------*/
using System;

namespace S.Models 
{

    /// <summary>
    ///  C# - Model Class - S.S
    ///  2019-07-27 11:01:04
    /// </summary>
    public class S {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   S myS = new S();
       ///   </code>
       /// </example>
        public S() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   S myS = new S( val1, val2,.. );
       ///   </code>
       /// </example>
        public S(String S) {
            this.S = S;
        }


        /**
        * Properties
        */

        public String S { get; set; }



        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}

File: S_20190727.vb
'--  
'-  DB Script Tool
'-  VB.Net (Visual Basic .Net) - 2019-07-27 11:04:04
'-  
'-  MODEL CLASSES FOR S DATABASE
'--



' S.vb -------------------------------------
Imports System

Namespace S.model 

' <summary>
' VB.Net - Model Class - S.S
' 2019-07-27 11:01:04
' </summary>
Public Class S 

    '--
    '- Constructor
    '- 
    '- Example: 
    '- Dim myS As New S 
    '--
    Public Sub New()
        '--
    End Sub

    '--
    '- Constructor
    '- 
    '- Example: 
    '- Dim myS As New S( val1, val2,.. )
    '--
    Public Sub New(ByVal S As String)
        Me.S = S
    End Sub


    '--
    '- Properties
    '--

    Public Property S As String


    '--
    '- Methods
    '--

    Public Overrides Function ToString() As String
        Return ""
    End Function

End Class

End Namespace